plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/N4jscOptions.java

```diff
@@ -243,7 +243,7 @@ void setDir(File file) {
 				// not implemented:
 				// @SubCommand(name = "watch", impl = WatchOptions.class),
 				// @SubCommand(name = "api", impl = APIOptions.class),
-				@SubCommand(name = "set-versions", impl = SetVersionsOptions.class),
+				@SubCommand(name = "setversions", impl = SetVersionsOptions.class),
```

Comment:
That hyphen made problems at several locations in Java and N4JS code. That is why I changed it. Maybe we can change it later since that would be nice, I agree.